app/assets/stylesheets/recipe_new.scss

```diff
@@ -7,6 +7,42 @@
   color: $slate;
 }
 
+.recipe-form-header {
+  font-family: 'Lobster', cursive;
+  font-size: 20pt;
+}
+
+.form-button {
+  background-color: $light-green;
+  color: $black;
+
+  &:hover {
+    background-color: $slate;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.